test/Generator/fixtures/expected/GenerateTypesMethodsTrait.php

```diff
@@ -477,7 +477,7 @@ public function setArray($array)
      *
      * @return object
      */
-    public function getObject()
+    public function getObject(): object
```

Comment:
https://wiki.php.net/rfc/object-typehint

`object` is a php 7.2 typehint

Reply:
Ok, excluded for now.